kommunicate/src/main/java/com/applozic/mobicomkit/api/conversation/AlTotalUnreadCountTask.java

```diff
@@ -27,12 +39,16 @@ public AlTotalUnreadCountTask(Context context, TaskListener callback) {
     @Override
     protected Integer doInBackground() {
         try {
-            // Call the List api method only if server call for list was not done before and return the unread count.
-            if (!ApplozicClient.getInstance(ApplozicService.getContextFromWeak(weakReferenceContext)).wasServerCallDoneBefore(null, null, null)) {
-                if (!Utils.isInternetAvailable(ApplozicService.getContextFromWeak(weakReferenceContext))) {
-                    return null;
+            String data = new MessageClientService(ApplozicService.getContextFromWeak(weakReferenceContext)).getMessages(null,null,null,null,null,false);
+            JsonObject jsonObject = JsonParser.parseString(data).getAsJsonObject();
```

Comment:
name of the field is vague here, change it to something which is more aligned to its usability

Reply:
fixed